cockatrice/src/dlg_connect.cpp

```diff
@@ -69,6 +71,19 @@ DlgConnect::DlgConnect(QWidget *parent)
 
     connect(savePasswordCheckBox, SIGNAL(stateChanged(int)), this, SLOT(passwordSaved(int)));
 
+	btnForgotPassword = new QPushButton(tr("Forgot password"));
+	btnForgotPassword->setFixedWidth(100);
```

Comment:
Tabs + the FixedWidth looks bad on larger screens. The button will need to be larger I feel.

<img width="443" alt="screenshot 2017-01-23 15 33 49" src="https://cloud.githubusercontent.com/assets/7460172/22221751/484cc3a6-e182-11e6-943d-dc63a6918a7a.png">